### PR TITLE
Add blogpost on grpc c# future

### PR DESCRIPTION
**Pull request by jtattermusch**
Preview: https://deploy-preview-759--grpc-io.netlify.app/blog/grpc-csharp-future/

### CODE REVIEW

**Comment by jtattermusch** [comment]
@chalin  can you please review from techwriter perspective?

**Comment by jtattermusch** [comment]
@JamesNK  @JunTaoLuo

**Comment by chalin** [comment]
@jtattermusch - could you rebase your PR? When I look at the preview, it is over the pre-docsy version of the site. I'll still review the content of the new blog post, but it would be preferable to see the post in it's true setting (docsy-themed blog section).

**Comment by jtattermusch** [comment]
> @jtattermusch - could you rebase your PR? When I look at the preview, it is over the pre-docsy version of the site. I'll still review the content of the new blog post, but it would be preferable to see the post in it's true setting (docsy-themed blog section).

Sure. At first I was confused but then I remembered that we now we have the "main" branch instead of "master" which explains why the PR was so out of date. Rebased.

**Review by chalin — CHANGES_REQUESTED** [review]
@jtattermusch - overall, I think this is ok, though I see opportunities for streamlining the text (there seems to be quite a bit of repetition / reformulation).

Here is what I propose to make it easier for me to offer you feedback and make copyedits to the text:

**Mark the blog entry as a draft** (for now) -- that is, **add `draft: true` to the front matter**. That'll allow us to merge the PR without the blog entry getting published to the production site. I'll then be able to make a pass over it, which I'll submit as separate PRs for your approval.

How does that sound?

**Comment by chalin** [comment]
Ah! Thanks for the rebase!

**Comment by jtattermusch** [comment]
I marked the blogpost as `draft: true`

**Comment by chalin** [comment]
> I marked the blogpost as `draft: true`

Thanks. Are you ok with the proposed plan? If so, I'll merge this PR.

**Comment by jtattermusch** [comment]
> > I marked the blogpost as `draft: true`
> 
> Thanks. Are you ok with the proposed plan? If so, I'll merge this PR.

Yes, sounds good. t

File: content/en/blog/grpc-csharp-future.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+From now on, we are no longer going to provide new features or enhancements for Grpc.Core. Important bugs and security issues will continue to be addressed in a normal way.
+
+We will publish Grpc.Core releases normally, with the usual 6 weekly cadence.
+
+The releases will be based on the newest grpc C core native library build, so all new features that don't require C# specific work will also be included.
+
+**Phase 2: Grpc.Core becomes "Deprecated"**
+
+**When: 1 year from now (May 2022)**
```
Comment (JamesNK) [inline content/en/blog/grpc-csharp-future.md:64]: One year feels like a short amount of time. For .NET Framework users especially there isn't a GA version of Windows available that supports Grpc.Net.Client yet.

I think some more time between maintenance and deprecated would be a good.

Comment (chalin) [reply content/en/blog/grpc-csharp-future.md:64]: @JamesNK - I'll let @jtattermusch respond to your comment. I wanted to reassure you that while I've merged the PR, the blogpost won't show up on the website until we remove the draft status -- which will be after round(s) of edits :)

Comment (jtattermusch) [reply content/en/blog/grpc-csharp-future.md:64]: We discussed this offline so I'm going to mark the conversation as resolved.

**Pull request merged**